### DISCUSSION TITLE
Failed to Install in Windows

### DISCUSSION THREAD

**Discussion by arc-nasc** [requarks/wiki#3759] [Help / Questions] [answered]
Hi!

I have failed installing wiki.js in windows.
I have all the systems requirements ok (postgres and node).

In the last step of the installation process, one is supposed to type in the powershell: `node server`.
I get the following error:

internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'C:\windows\system32\server'
←[90m    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)←[39m
←[90m    at Function.Module._load (internal/modules/cjs/loader.js:725:27)←[39m
←[90m    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)←[39m
←[90m    at internal/main/run_main_module.js:17:47←[39m {
  code: ←[32m'MODULE_NOT_FOUND'←[39m,
  requireStack: []
}

Can anyone help me with this?

Thanks!

**Comment by NGPixel** [comment, marked as answer]
That's because you're not in the correct directory... You need to open a powershell prompt inside the folder where you installed Wiki.js.

**Comment by arc-nasc** [comment]
Thank you for your message.

I have run the <node server> in C:\wiki, which is where it is currently
installed. The same error occured.

Was there another directory that I should go to?



On Wed, 24 Mar 2021 at 20:43, Nicolas Giard ***@***.***>
wrote:

> That's because you're not in the correct directory... You need to open a
> powershell prompt inside the folder where you installed Wiki.js.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Requarks/wiki/discussions/3759#discussioncomment-527460>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMYCZCVUMQYFPGG3MLMGXQLTFKBKHANCNFSM4ZXU4DTQ>
> .
>


-- 

*Alexandre Nascimento*
